src/test/java/bio/terra/service/dataset/flight/ingest/IngestUtilsTest.java

```diff
@@ -43,6 +43,17 @@ public void testParseValidBlobURL() {
         equalTo("test/azure-simple-dataset-ingest-request.csv"));
   }
 
+  public void testValidURLWithSpecialCharaterBlobPaths() {
+    IngestUtils.validateBlobAzureBlobFileURL(
+        "https://tdrconnectedsrc1.blob.core.windows.net/synapsetestdata/test/azure_simple_dataset_ingest_request.csv");
```

Comment:
Minor suggestion: if you replace `https://tdrconnectedsrc1.blob.core.windows.net/synapsetestdata/test/` with a variable I think it would be clearer that the difference between these urls is just the blob name.

Reply:
Fair enough.  I never know with tests if it's helpful to see the whole path written out.  but happy to change it